db/schema.rb

```diff
@@ -17,15 +17,15 @@
   enable_extension "plpgsql"
 
   create_table "addresses", force: :cascade do |t|
-    t.string   "flat",        limit: 255
-    t.string   "street",      limit: 255
-    t.string   "postal_code", limit: 255
```

Comment:
I don't know how to fix this. I wonder if there is a setting in my postgres that doesn't assign default limits when setting up a database? These limits are being added despite them not being specified in the migration.